pkg/clusterrouter/proxy/proxy_server.go

```diff
@@ -11,10 +11,9 @@ import (
 	"strings"
 	"sync"
 
-	v32 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
-
 	"github.com/rancher/norman/httperror"
-	factory "github.com/rancher/rancher/pkg/dialer"
+	v32 "github.com/rancher/rancher/pkg/apis/management.cattle.io/v3"
+	dialer2 "github.com/rancher/rancher/pkg/dialer"
```

Comment:
Is this changing because we use the factory var in other places ? Like arg to NewRemote

Reply:
No, I'm afraid the reason is much simpler than that: that is what Goland chose when I moved everything back.

Reply:
It's because `	"github.com/rancher/rancher/pkg/types/config/dialer"` is already imported